packages/apollo-gateway/src/datasources/__tests__/RemoteGraphQLDataSource.test.ts

```diff
@@ -115,6 +119,50 @@ describe('willSendRequest', () => {
   });
 });
 
+describe('didReceiveResponse', () => {
```

Comment:
Can we add another test that tests the use of `didReceiveResponse` without the `context` parameter being passed (the previous behavior)?

Reply:
See related: https://github.com/apollographql/apollo-server/pull/3360/files/968c6b8c5894de08d5c6801b3f7385ebd300695f#r330680535